src/mme-app/mmeStates/attachWfMbResp.cpp

```diff
@@ -63,4 +63,13 @@ void AttachWfMbResp::initialize()
                 actionTable.addAction(&ActionHandlers::attach_done);
                 eventToActionsMap.insert(pair<uint16_t, ActionTable>(MB_RESP_FROM_SGW, actionTable));
         }
+        {
+                ActionTable actionTable;
+                actionTable.addAction(&ActionHandlers::handle_attach_request);
+                actionTable.addAction(&ActionHandlers::del_session_req);
+                actionTable.addAction(&ActionHandlers::send_s1_rel_cmd_to_ue);
```

Comment:
doing S1 release might be of no use cause already Attach has come on new ENB S1apID.